<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSoundPropertyWidget</class>
 <widget class="QWidget" name="CSoundPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_title"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_artist">
     <property name="text">
      <string>Artist</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_artist"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_album">
     <property name="text">
      <string>Album</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="lineEdit_album"/>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_tracknumber"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_date"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_genre"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_genre">
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_tracknumber">
       <property name="text">
        <string>Track number</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_software">
     <property name="text">
      <string>Software</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lineEdit_software"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_copyright">
     <property name="text">
      <string>Copyright</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="lineEdit_copyright"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_license">
     <property name="text">
      <string>License</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="lineEdit_license"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_comment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_comment"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
